hardhat.config.ts

```diff
@@ -23,30 +25,46 @@ const config: HardhatUserConfig = {
     ],
   },
   paths: {
-    sources: './contracts',
-    tests: './test',
-    cache: './cache',
-    artifacts: './artifacts',
+    sources: "./contracts",
+    tests: "./test",
+    cache: "./cache",
+    artifacts: "./artifacts",
   },
   typechain: {
-    outDir: 'typechain',
+    outDir: "typechain",
   },
   mocha: {
     timeout: 5000000,
   },
   networks: {
     mainnet: {
-      url: 'https://mainnet.infura.io/v3/97e75e0bbc6a4419a5dd7fe4a518b917',
+      url: "https://mainnet.infura.io/v3/97e75e0bbc6a4419a5dd7fe4a518b917",
       gasPrice: 80000000000,
     },
     goerli: {
-      url: 'https://goerli.infura.io/v3/77c3d733140f4c12a77699e24cb30c27',
+      url: "https://goerli.infura.io/v3/77c3d733140f4c12a77699e24cb30c27",
       timeout: 10000000,
     },
+    devnet: {
+      // Add current URL that you spawned if not using automated spawning
+      url: `${process.env.DEVNET_RPC_URL}`,
+      chainId: 1,
+    },
   },
   etherscan: {
     apiKey: `${process.env.ETHERSCAN_API_KEY}`,
   },
+  tenderly: {
+    project: `${process.env.TENDERLY_PROJECT_SLUG}`,
+    username: `${process.env.TENDERLY_ACCOUNT_ID}`,
+  },
 };
 
+// These are needed for tenderly verification to be automatic,
+// they don't seem to work with DevNet for some reason,
+// however Tenderly operations do not work if we don't call
+// this here, it is required for integration with Hardhat.
+// !!! This should be uncommented when using Tenderly DevNet !!!
+// tenderly.setup({ automaticVerifications: true });
```

Comment:
Not sure I fully understand this comment

"These don't work for tenderly DevNet"

but also

"Tenderly operations don't work if we don't include this, uncomment when using Tenderly DevNet"?